tests/foreman/ui/test_ansible.py

```diff
@@ -25,34 +25,6 @@
 from robottelo.config import settings
 
 
-def test_positive_import_all_roles(target_sat):
```

Comment:
You could have used fixtures that creates a new satellite where you could have import the roles etc instead of going to the destructive satellite testing.

But ultimately even destructive tests are using new satellite so one and all its the same.

Reply:
yeah